src/main/java/home/automation/LightsWrapper.java

```diff
@@ -0,0 +1,20 @@
+package home.automation;
+
+public class LightsWrapper implements IDevice{
+    Lights lights;
+
+    public LightsWrapper(Lights lights){
+        this.lights=lights;
+    }
+
+    @Override
+    public void turnOn() {
+        lights.dimPercent(50);
```

Comment:
This way I can't setup a light to be fully bright when turning it on; if this were a settable class field this could be possible.